Examples/SubscriberExample/Sources/Utils/LocationAnimator/LocationAnimator.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+import AblyAssetTrackingCore
+import Combine
+
+protocol LocationAnimator {
+    /**
+     Time interval (in seconds) for publishing `fragmentaryPosition`.
+     Default value is 5.0 seconds
+     */
+    var fragmentaryPositionInterval: TimeInterval { get set }
+    
+    /**
+     This closure returns position in CADisplayLink framerate
+     */
+    func trackablePosition(_ closure: @escaping (Position) -> Void)
+    
+    /**
+     This closure returns trackable position every `fragmentaryPositionInterval`
+     */
+    func fragmentaryPosition(_ closure: @escaping (Position) -> Void)
```

Comment:
I've meant the "some" position from `animation positions array`

Reply:
Perhaps I'd call them `addFrequentlyUpdatingPositionSubscriber` and `addInfrequentlyUpdatingPositionSubscriber` and document examples of what each is meant to be used for, what do you think?

Reply:
I'm not sure if the prefix `add` is appropriate here, because it could suggest that we can `add` more than one subscriber for this event, and this is not true.
My proposal is `subscriberForFrequentlyUpdatingPosition` and similarly, `subscriberForInfrequentlyUpdatingPosition`. It also will be consistent with this https://github.com/ably/ably-asset-tracking-swift/blob/b427c8c3cf8d52cdad20a662e49d4761a6148d0a/Sources/AblyAssetTrackingInternal/AblyWrapper/AblySubscriber.swift#L87

Reply:
That's a good point. I suggested `add` because I wanted there to be a verb there - perhaps `subscribeForFrequentlyUpdatingPosition` etc? That will be more consistent with what you've quoted above.

Reply:
changed 06d2e9e25cd6307e467ced029ff1f859035baa49

Reply:
Actually, not sure why I didn't think of this before, but why don't we be more explicit in the naming - `setAnnotationPositionSubscriber` and `setCameraPositionSubscriber`?